src/main/java/com/adaptavist/tm4j/jenkins/extensions/FormInstance.java

```diff
@@ -0,0 +1,85 @@
+package com.adaptavist.tm4j.jenkins.extensions;
+
+import hudson.model.Describable;
+import hudson.model.Descriptor;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.io.Serializable;
+
+public class FormInstance implements Serializable, Describable<FormInstance> {
```

Comment:
It seems strange for me to have one mixed class... I suppose that we will never have a cloudAddress and a serverAddress